applications/matter_bridge/README.rst

```diff
@@ -280,6 +302,15 @@ You can enable the :ref:`matter_bridge_app_bridged_support` by using the followi
 * :kconfig:option:`CONFIG_BRIDGED_DEVICE_SIMULATED` - For the simulated bridged device.
 * :kconfig:option:`CONFIG_BRIDGED_DEVICE_BT` - For the Bluetooth LE bridged device.
 
+The simulated OnOff Light bridged device can operate in the following modes:
+
+* Autonomous - The simulated device periodically changes its state.
+* Controllable - The user can explicitly control the On/Off state with shell command.
+
+To build the simulated OnOff Light data provider in autonomous mode, select :kconfig:option:`CONFIG_BRIDGED_DEVICE_SIMULATED_ONOFF_AUTOMATIC` Kconfig option.
+To build the simulated OnOff Light data provider in controllable mode, select :kconfig:option:`CONFIG_BRIDGED_DEVICE_SIMULATED_ONOFF_SHELL` Kconfig options.
+The :kconfig:option:`CONFIG_BRIDGED_DEVICE_SIMULATED_ONOFF_SHELL` is enabled by default.
```

Comment:
```suggestion
* Autonomous - The simulated device periodically changes its state.
  To build the simulated OnOff Light data provider in this mode, select the :kconfig:option:`CONFIG_BRIDGED_DEVICE_SIMULATED_ONOFF_AUTOMATIC` Kconfig option.
* Controllable - The user can explicitly control the On/Off state with shell commands.
  To build the simulated OnOff Light data provider in this mode, select the :kconfig:option:`CONFIG_BRIDGED_DEVICE_SIMULATED_ONOFF_SHELL` Kconfig option.
  This is enabled by default.
```
Since we're only dealing with one device type now, might as well consolidate the information.